src/fshtypes/FshQuantity.ts

```diff
@@ -9,7 +9,10 @@ export class FshQuantity extends FshEntity {
 
   toString(): string {
     let str = this.value.toString();
-    if (this.unit?.code != null) str += ` '${this.unit.code}'`;
+    if (this.unit?.code != null) {
+      str += ` '${this.unit.code}'`;
+      if (this.unit?.display != null) str += ` "${this.unit.display}"`;
```

Comment:
Very minor, but I think at this point, `this.unit` is guaranteed to be non-null, so the `?` isn't necessary.

Reply:
Ah good point! I moved that around after I wrote it and didn't look very closely at it. I'll update!

Reply:
Should be addressed in 1c6738f.